fe/fe-core/src/main/java/com/starrocks/analysis/ShowAuthenticationStmt.java

```diff
@@ -0,0 +1,59 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.ScalarType;
+import com.starrocks.qe.ShowResultSetMetaData;
+import com.starrocks.sql.ast.AstVisitor;
+
+public class ShowAuthenticationStmt extends ShowStmt {
```

Comment:
authorization not authentication

Reply:
Authentication (from [Greek](https://en.wikipedia.org/wiki/Greek_language): αὐθεντικός authentikos, "real, genuine", from αὐθέντης authentes, "author") is the act of [proving](https://en.wikipedia.org/wiki/Proof_(truth)) an [assertion](https://en.wikipedia.org/wiki/Logical_assertion), such as the [identity](https://en.wikipedia.org/wiki/Digital_identity) of a computer system user.